machAeroTutorials/aero_adflow_polar.rst

```diff
@@ -42,9 +42,9 @@ Import libraries
 As in the previous example, we first import ADflow.
 We also need to import :doc:`baseclasses <baseclasses:index>`, which is a library of problem and solver classes used to encourage a common API within the MACH suite.
```

Comment:
I know this is not part of the edits, I just think this is a bit of a repetition from the previous page?

Reply:
I think this is probably redundant. The text here is almost verbatim from what is done in `aero_adflow.rst`. How about we just remove and cite the one in `aero_adflow.rst`. The `aeroOptions` dictionary I would say is also redundant.